cmd/terminal.go

```diff
@@ -70,9 +70,14 @@ const (
 )
 
 var terminalCmd = &cobra.Command{
-	Use:   "terminal DEVICE_ID",
+	Use:   "terminal [DEVICE_ID]",
 	Short: "Access a device's remote terminal",
-	Args:  cobra.RangeArgs(0, 1),
+	Long: "Access a device's remote terminal\n" +
```

Comment:
technically speaking `'s` is reserved for persons, right? maybe would be better to rephrase? "Remotely access a terminal on a device"? unsure.

Reply:
I like `Remotely access a terminal on a device` too

Reply:
:+1: